apps/shipments/serializers.py

```diff
@@ -601,9 +601,24 @@ def data(self):
         return self.datetime_filters(queryset_diff)
 
 
+class ShipmentOverviewTransitState(Enum):
+    """
+    We need this enum here since enum classes cannot being subclassed
+    and AWAITING_PICKUP is not a valid state for this endpoint
```

Comment:
Instead of duplicating this enum, please add a `validate_state` method to DevicesQueryParamsSerializer that validates != AWAITING_PICKUP